widget/embedded/src/components/Quote/QuoteTrigger .tsx

```diff
@@ -0,0 +1,161 @@
+import type { QuoteTriggerProps } from './Quote.types';
+import type { ReactNode } from 'react';
+
+import { i18n } from '@lingui/core';
+import {
+  ChevronDownIcon,
+  ChevronRightIcon,
+  Divider,
+  Image,
+  Tooltip,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import { getContainer } from '../../utils/common';
+
+import {
+  IconContainer,
+  ImageContainer,
+  MoreStep,
+  Row,
+  Trigger,
+} from './Quote.styles';
+
+const MAX_STEPS = 4;
+const INDEX_OF_STEP = 3;
+
+const ImageComponent = (props: {
+  content: ReactNode;
+  state?: 'error' | 'warning' | undefined;
+  src: string;
+  open?: boolean;
+  className?: string;
+}) => {
```

Comment:
NIT: Use Destructuring assignment outside the function argument:

like this: const { content, state, src, open, className } = props;